code/data_functions.py

```diff
@@ -271,9 +254,86 @@ def clean_IRENA_regression(param, paths):
                                                IRENA_summary.loc[(country, 'Concentrated solar power'), 'FLH (h)']]
         except KeyError:
             print(country)
-            import pdb; pdb.set_trace()
             
     IRENA_regression.to_csv(paths['IRENA_regression'], sep=';', decimal=',', index=True)
+    print("files saved: " + paths['IRENA_regression'])
+
+
+def clean_TS_regression(param, paths):
+    """
```

Comment:
so this code only creates TS for the technologies currently in *technologies*?
I would prefer to save them in four separate files, so that they aren't overwritten.